src/ngraph_builder.cc

```diff
@@ -906,11 +907,34 @@ static tf::Status TranslateFillOp(const tf::Node* op,
     ng_output_shape[i] = dims_vec[i];
     ng_axis_set.insert(i);
   }
-  SaveNgOp(ng_op_map, op->name(), make_shared<ng::op::Broadcast>(
-                                      ng_value, ng_output_shape, ng_axis_set));
+  SaveNgOp(
+      ng_op_map, op->name(),
+      make_shared<ng::op::Broadcast>(ng_value, ng_output_shape, ng_axis_set));
   return tf::Status::OK();
 }
 
+auto ng_floordiv = [](std::shared_ptr<ng::Node> ng_input1,
```

Comment:
Shouldn't define this lambda outside of the FloorDivOp(...). The reason you defined it here so that you can use them in the two Floor ops below but I would refactor it the following way:

1. Move the lambda `ng_floordiv` inside the TranslateFloorDivOp and use it there. 
2. In  TranslateFloorModOp() build the nGraph op in lines 931 through line 933 instead of calling the `ng_floordiv` lambda. Then return the resulting op.

Reply:
@avijit-nervana  Agreed. However, our current design does not allow us to reuse an already implemented op-translation. For simpler ops, it is not much of a replication but there could be other ops where this might amount to significant code replication. I had faced a similar situation, while implementing another op, there too it was 1-2 line of replication. We might have to re-design, if we come across an op which uses 2-3 simpler ops.